irohad/consensus/yac/impl/peer_orderer_impl.cpp

```diff
@@ -19,23 +19,14 @@ namespace iroha {
           std::shared_ptr<ametsuchi::PeerQueryFactory> peer_query_factory)
           : peer_query_factory_(peer_query_factory) {}
 
-      boost::optional<ClusterOrdering> PeerOrdererImpl::getInitialOrdering() {
-        return peer_query_factory_->createPeerQuery() |
-            [](const auto &query) { return query->getLedgerPeers(); } |
-            [](const auto &peers) { return ClusterOrdering::create(peers); };
-      }
-
       boost::optional<ClusterOrdering> PeerOrdererImpl::getOrdering(
-          const YacHash &hash) {
-        return peer_query_factory_->createPeerQuery() |
-            [](const auto &query) { return query->getLedgerPeers(); } |
-            [&hash](auto peers) {
-              std::seed_seq seed(hash.vote_hashes.block_hash.begin(),
-                                 hash.vote_hashes.block_hash.end());
-              std::default_random_engine gen(seed);
-              std::shuffle(peers.begin(), peers.end(), gen);
-              return ClusterOrdering::create(peers);
-            };
+          const YacHash &hash,
+          std::vector<std::shared_ptr<shared_model::interface::Peer>> peers) {
+        std::seed_seq seed(hash.vote_hashes.block_hash.begin(),
+                           hash.vote_hashes.block_hash.end());
+        std::default_random_engine gen(seed);
+        std::shuffle(peers.begin(), peers.end(), gen);
```

Comment:
Please reply me, why we use shuffle? What is the end goal?

Reply:
https://github.com/hyperledger/iroha/blob/536d4043e1d985e8bc21edcd75f19987930f2a9a/irohad/consensus/yac/yac_peer_orderer.hpp#L24
So it is used to generate an order based on hash. Could you suggest a comment to add to the code?